apps/condo/domains/banking/schema/BankAccount.test.js

```diff
@@ -62,6 +73,20 @@ describe('BankAccount', () => {
                 expect(bankAccount.currencyCode).toEqual('RUB')
             })
 
+            test('service can', async () => {
```

Comment:
Could you please create 2 test cases:
1. When service use can, having organization context
2. When cannot, not having them (expect access denied exception)